test/Fail/Issue3983.err

```diff
@@ -1,4 +1,5 @@
 Issue3983.agda:23,1-26,8
+warning: -W[no]GenericUseless
```

Comment:
TODO (maybe some other PR): Make a specific warning for this rather than `GenericUseless`.

(Btw., this is a warning you would not want to ignore.)